### PR TITLE
Create viewer page

### PR DESCRIPTION
**Pull request by jtomeck**
@jbeezley I'm having some trouble finding the best way to add the controls to each of the panels. I have each panel set up as expansion panels that are created in `data() { return { ... } }`. I think this is the right way to go, but I don't think I have the knowledge to add the inputs and controls for each in this manner. Let me know if this is confusing the way I'm explaining it or if you'd like to have a hangout to talk it through.

### CODE REVIEW

File: src/pages/ViewerPage.vue [hunk 1]
```diff
@@ -1,18 +1,145 @@
...
 export default {
   name: 'ViewerPage',
-  components: {
-  },
+  components: {},
   data() {
-    return {};
+    return {
+      panel: [],
+      items: 3, // Must be equal to number of items in viewPanels array
```
Comment (jbeezley) [inline src/pages/ViewerPage.vue:118]: Remove

File: src/pages/ViewerPage.vue [hunk 2]
```diff
@@ -1,18 +1,145 @@
...
+          header: 'Dynamic Properties',
+        },
+      ],
+    };
   },
   methods: {
+    // Create an array the length of our items
+    // with all values as true
+    all() {
+      this.panel = [...Array(this.items).keys()].map((k, i) => i);
```
Comment (jbeezley) [inline src/pages/ViewerPage.vue:137]:
```suggestion
      this.panel = [...Array(this.viewPanel.length).keys()];
```

File: src/pages/ViewerPage.vue [hunk 3]
```diff
@@ -1,18 +1,145 @@
...
+              <div class="d-flex panel-footer pt-3">
+                <v-spacer />
+                <v-btn
+                  color="primary"
+                  small
+                >
+                  Apply
+                </v-btn>
+              </div>
+            </v-expansion-panel-content>
```
Comment (jbeezley) [inline src/pages/ViewerPage.vue:102]: We'll probably want make components for the contents of these controls, but this is fine for now.

**Review by jbeezley — CHANGES_REQUESTED** [review]

File: src/pages/ViewerPage.vue (hunk 3 above)
Comment (jtomeck) [reply src/pages/ViewerPage.vue:102]: I'm completely onboard with this approach. I wanted to see if there was the ability to put `<componentName />` in the array for controls, but you need to also register that component and a couple errors are thrown. Not sure the best way to handle this.

**Comment by jtomeck** [comment]
@jbeezley - The suggested edits, break the expand all button. I pushed them so you can take a look at whether I implemented them properly or not.

**Pull request merged**